x/wasm/ibctesting/chain.go

```diff
@@ -497,8 +497,8 @@ func (chain *TestChain) CreateTMClientHeader(chainID string, blockHeight int64,
 	// MakeCommit expects a signer array in the same order as the validator array.
 	// Thus we iterate over the ordered validator set and construct a signer array
 	// from the signer map in the same order.
-	signerArr := make([]tmtypes.PrivValidator, len(tmValSet.Validators))
-	for i, v := range tmValSet.Validators {
+	signerArr := make([]tmtypes.PrivValidator, len(tmValSet.Validators)) //nolint:staticcheck // TODO: determine if there's really a possible nil pointer dereference
```

Comment:
@faddat Are you going to follow up on this? Let's not add more todos in code

Reply:
PS: wdyt on this one?

Now that I'm looking at it, the TODO is def because I wasn't real sure at all.